kontonummer.js

```diff
@@ -267,13 +276,13 @@
             control  : 10
         }
     },{
-        name    : 'Royal Bank of Scotland',
-        regex   : /^(909[0-9])/,
+        name    : 'Riksgälden',
+        regex   : /^(988[0-9])/,
```

Comment:
For this clearing number I think the control number should be 11 according to the docs. Riksgälden exist for two different interval of clearing numbers.

Reply:
You are correct. I got confused by what `control` means in this context.

Reply:
I've fixed this in the fixup commit feeaf554d2f156f6c1a3bacf43f44653ec7ba04b